### ISSUE TITLE
Contrast Minimum Accessibility

### ISSUE THREAD

**Issue by jolanglinais** [accordproject/template-studio-v2#42]
# WCAG 2.0 Level AA [1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) (Contrast (Minimum))

## Tasks
- Contrast ratio should be at least `4.5:1`.

## Related Issues: 
- Issue [#57](https://github.com/accordproject/cicero-ui/issues/57) in Cicero-UI

**Comment by Sweta271097** [comment]
@irmerk Can I take up this issue?

**Comment by jolanglinais** [comment]
Moving this back to open after 7 days.

**Comment by CDharmateja** [comment]
Can I work on this issue?
